web/src/app/abstract-directives/list.directive.ts

```diff
@@ -56,7 +56,9 @@ export abstract class ListDirective extends BaseDirective implements OnInit, OnD
       const list = data.results;
       this.data = new MatTableDataSource<any>(list);
       this.changeCount(data.count);
-      this.listItemEvt.emit({ cmd: 'onLoad', row: list[0] });
+      if (Array.isArray(list)) {
+        this.listItemEvt.emit({ cmd: 'onLoad', row: list[0] });
```

Comment:
is allow undefined value? for row

Reply:
It seems that such a check is quite enough to fix the error.